index.html

```diff
@@ -4344,6 +4344,23 @@ <h1>DID Methods</h1>
             <a href="https://workday.github.io/work-did-method-spec/">Workday DID Method</a>
           </td>
         </tr>
+       <tr>
+          <td>
+            did:snplab:
```

Comment:
Move to sorted order.

Reply:
Thanks for pointing out. I made the change.

Reply:
It's still at the end.  The whole row should be in method name order up in the did:s* part.

Reply:
I have put the did:snplab  between did:sirius and did:sol.

Thanks.